src/app/common/stores/applications/SsvWeb/Operator.store.ts

```diff
@@ -209,20 +209,20 @@ class OperatorStore extends BaseStore {
   /**
    * Get max validators count
    */
-  async getOperatorValidatorsLimit(): Promise<number> {
+  async updateOperatorValidatorsLimit(): Promise<void> {
     const walletStore: WalletStore = this.getStore('Wallet');
     const contract: Contract = walletStore.getterContract;
     if (this.operatorValidatorsLimit === 0) {
       this.operatorValidatorsLimit = await contract.methods.getValidatorsPerOperatorLimit().call();
     }
-    return this.operatorValidatorsLimit;
   }
 
   /**
    * Check if operator registrable
    */
-  isOperatorRegistrable(validatorsRegisteredCount: number) {
-    return this.operatorValidatorsLimit > validatorsRegisteredCount;
+  isOperatorRegistrable(validatorsRegisteredCount: number): boolean {
+    const result = Math.min(validatorsRegisteredCount + config.GLOBAL_VARIABLE.OPERATOR_VALIDATORS_LIMIT_PRESERVE, this.operatorValidatorsLimit);
```

Comment:
Something here isn't clear to me.
there are 2 scenarios:
1. result = validatorsRegisteredCount + config.GLOBAL_VARIABLE.OPERATOR_VALIDATORS_LIMIT_PRESERVE
2.  result = this.operatorValidatorsLimit

in first case:
this.operatorValidatorsLimit > result will result in true.
in second case:
this.operatorValidatorsLimit > this.operatorValidatorsLimit is always false.

it seems that this is equivalent to just returning:
 this.operatorValidatorsLimit > validatorsRegisteredCount + config.GLOBAL_VARIABLE.OPERATOR_VALIDATORS_LIMIT_PRESERVE

Or you possibly meant to check something else?

Reply:
u right, it's equivalent to this.operatorValidatorsLimit > validatorsRegisteredCount + config.GLOBAL_VARIABLE.OPERATOR_VALIDATORS_LIMIT_PRESERVE

Reply:
@alexstolr @meshin-dev done